src/executor/execute.rs

```diff
@@ -57,6 +60,16 @@ pub enum Payload {
     Commit,
     #[cfg(feature = "transaction")]
     Rollback,
+
+    #[cfg(feature = "metadata")]
+    ShowVariable(PayloadVariable),
+}
```

Comment:
It has nothing to do with metadata, but how about attaching indexes to each other?

Reply:
I think... I don't get the point. Could you provide more details?

Reply:
This is the sorting story inside `ExecuteError`.
It's related to formatting, so I don't think you need to think too much about it.

Group them in the form below.
```rust
     #[cfg(feature = "index")]
     CreateIndex,
     #[cfg(feature = "index")]
     DropIndex,

     #[cfg(feature = "transaction")]
     StartTransaction,
     #[cfg(feature = "transaction")]
     Commit,
     #[cfg(feature = "transaction")]
     Rollback,

     #[cfg(feature = "metadata")]
     ShowVariable(PayloadVariable),
}
```